docs/CI-Jobs.md

```diff
@@ -0,0 +1,89 @@
+| ROSA Non STS                          | Latest Pre-GA          | [Test Grid][SD CICD ROSA]                           |
+| ROSA HCP                              | Latest GA              | [Test Grid][SD CICD ROSA HCP]                       |
+| ROSA BYOVPC Proxy Install             | Latest Pre-GA          | [Test Grid][SD CICD ROSA BYOVPC Proxy Install]      |
+| ROSA BYOVPC Proxy Post Install        | Latest Pre-GA          | [Test Grid][SD CICD ROSA BYOVPC Proxy Post Install] |
+| OSD AWS Upgrade                       | Latest Y-1 To Latest Y | [Test Grid][SD CICD OSD AWS Upgrade]                |
+| OSD AWS SREP Operator Informing Suite | Latest Pre-GA          | [Test Grid][SD CICD OSD AWS Informing Suite]        |
+
+The `ROSA HCP 'HyperShift'` job sends alerts to the following slack channel: [#sd-hypershift-info].
+
+### OCP Versions
+
+The [SD CICD Periodic Jobs](#sd-cicd-periodic-jobs) above may not always be
+testing the same OpenShift version. One of the main reason for this is in
+regards to ROSA STS jobs. STS policies may not exist yet for the latest pre-GA
+version. Which means the job needs to install the latest GA'd version,
+until the policies are enabled for the latest pre-GA version.
+
+| OCP Version            | Definition                                                                           |
+| ---------------------- | ------------------------------------------------------------------------------------ |
+| Latest GA              | Installs the latest GA X.Y.Z version from the channel group specified by the job     |
+| Latest Pre-GA          | Installs the latest Pre-GA X.Y.Z version from the channel group specified by the job |
+| Latest Y-1 To Latest Y | Installs the latest GA Y-1 version and upgrades to latest Y                          |
+
+## TRT Nightly Periodic Jobs
+
+The table below represents the periodic jobs that run to validate Managed
+OpenShift for new nightly OCP TRT builds and provide a *informing* signal
+to the release.
+
+| OCP Version | OSD AWS                       | OSD GCP                       | ROSA STS              | ROSA HCP |
+| ----------- | ----------------------------- | ----------------------------- | --------------------- | -------- |
+| 4.14        | [Test Grid][4.14 TRT OSD AWS] | [Test Grid][4.14 TRT OSD GCP] | Refer to [SDCICD-557] | -        |
+| 4.13        | [Test Grid][4.13 TRT OSD AWS] | [Test Grid][4.13 TRT OSD GCP] | Refer to [SDCICD-557] | -        |
+| 4.12        | [Test Grid][4.12 TRT OSD AWS] | [Test Grid][4.12 TRT OSD GCP] | Refer to [SDCICD-557] | -        |
+| 4.11        | [Test Grid][4.11 TRT OSD AWS] | [Test Grid][4.11 TRT OSD GCP] | Refer to [SDCICD-557] | -        |
+| 4.10        | [Test Grid][4.10 TRT OSD AWS] | [Test Grid][4.10 TRT OSD GCP] | Refer to [SDCICD-557] | -        |
+
+These jobs send alerts to the following slack channel: [#sd-cicd-alerts].
+
+[SD CICD Test Grid Dashboard]: https://testgrid.k8s.io/redhat-openshift-osd
```

Comment:
this block of text isn't rendering as markdown. I think the brackets are the problem

Reply:
ah ok. TIL :)